src/Item/Item.router.spec.ts

```diff
@@ -515,20 +515,22 @@ describe('Item router', () => {
               expect(Item.findByCollectionIds).toHaveBeenCalledWith(
                 [dbCollectionMock.id],
                 undefined,
+                undefined,
                 undefined
               )
             })
         })
       })
 
-      describe('and pagination params are passed', () => {
+      describe('and pagination and synced params are passed', () => {
         let baseUrl: string
         let page: number
         let limit: number
+        let synced: boolean
         beforeEach(() => {
-          ;(page = 1), (limit = 1)
+          ;(page = 1), (limit = 1), (synced = false)
```

Comment:
This could be a bit more clear removing the commas right?
```suggestion
          page = 1
          limit = 1
          synced = false
```